rust/datafusion/src/logicalplan.rs

```diff
@@ -1087,7 +1101,7 @@ impl LogicalPlanBuilder {
     /// Apply a projection
     pub fn project(&self, expr: Vec<Expr>) -> Result<Self> {
         let input_schema = self.plan.schema();
-        let projected_expr = if expr.contains(&Expr::Wildcard) {
```

Comment:
I had to drop partialEq for expressions, and therefore also dropped this if that was strictly not needed.